test/fuzz/fuzz_lib.zig

```diff
@@ -12,7 +12,7 @@ const Parser = aro.Parser;
 var fixed_buffer_mem: [10 * 1024 * 1024]u8 = undefined;
 
 export fn compile_c_buf(buf: [*]const u8, len: c_int) void {
-    compileSlice(buf[0..@intCast(usize, len)]) catch unreachable;
+    compileSlice(buf[0..@as(usize, @intCast(len))]) catch unreachable;
```

Comment:
Doesn't seem like `@as` should be needed.

Reply:
Removed. This file has bitrotted a bit; I haven't run the fuzzer in a while.